falcon/util/uri.py

```diff
@@ -161,6 +168,8 @@ def encoder(uri):
     uri (str): URI fragment to encode. It is assumed not to cross delimiter
         boundaries, and so any reserved URI delimiter characters
         included in it will be percent-encoded.
+    check_is_escaped (boolean): Indicates a uri may have already been escaped
+        and should be treated as such
```

Comment:
Full stop (period) missing at the end of the sentence.

Reply:
Don't forget to mention the default value.